ci/kokoro/Dockerfile.ubuntu

```diff
@@ -0,0 +1,81 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+ARG DISTRO_VERSION=18.04
+FROM ubuntu:${DISTRO_VERSION}
+
+RUN apt update && \
+    apt install -y \
+        build-essential \
+        clang \
+        cmake \
+        curl \
+        doxygen \
+        gawk \
+        git \
+        gcc \
+        golang \
+        g++ \
+        libc-ares-dev \
+        libc-ares2 \
+        libssl-dev \
+        make \
+        ninja-build \
+        pkg-config \
+        python-pip \
+        shellcheck \
+        tar \
+        unzip \
+        wget \
+        zlib1g-dev
+
+# Install newer c-ares on Ubuntu 16.04.
```

Comment:
Not for this PR, should we split the Dockerfile for Ubuntu 16.04 vs. 18.04?

Reply:
Agreed. Filed #5